src/MagnumPlugins/WebPImporter/WebPImporter.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
+                2020, 2021, 2022 Vladimír Vondruš <mosra@centrum.cz>
```

Comment:
You could also add yourself here (and in other files you changed)

Reply:
I have added myself in the files where I did substantial changes and files directly related to WebPImporter. But I ignored the files which already existed, where I only did very small changes (e.g. renaming the importer, like "webp" instead of "png") and CMakeLists / configuration files. I wanted to ask before making additions to them too.